cpp/src/hdbscan/detail/utils.h

```diff
@@ -25,13 +25,16 @@
 
 #include <cuml/cluster/hdbscan.hpp>
 
+#include <raft/core/device_mdspan.hpp>
 #include <raft/label/classlabels.cuh>
 #include <raft/linalg/matrix_vector_op.cuh>
 #include <raft/linalg/norm.cuh>
+#include <raft/matrix/math.cuh>
 
 #include <algorithm>
 
 #include "../condensed_hierarchy.cu"
+#include <common/fast_int_div.cuh>
 
 #include <thrust/copy.h>
 #include <thrust/execution_policy.h>
```

Comment:
We should eventually move `Utils::normalize` into RAFT. These are very useful functions. We have functions for norms, but normalization is just as important and pops up just as frequently.